clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -126,7 +126,7 @@ public int read() throws IOException {
       try {
         mCurrentCacheStream.write(data);
       } catch (IOException ioe) {
-        LOG.warn("Block of ID {} could not be cached into Tachyon. Exception: {}",
+        LOG.warn("The blockId {} could not be cached into Tachyon. Exception: {}",
```

Comment:
For a user-facing message like this, it's better to use regular words, e.g. "The block with ID {} could not be ..."